diff.py

```diff
@@ -255,6 +255,20 @@ def complete_symbol(
         help="""Automatically update when source/object files change.
         Recommended in combination with -m.""",
     )
+    parser.add_argument(
+        "-0",
+        "--zeroway",
```

Comment:
The term zeroway is a bit silly, maybe --oneway-base? The term oneway is also misleading and would imo be good to replace with something else, though I don't have a great suggestion off hand.